### DISCUSSION TITLE
What do you guys use?

### DISCUSSION THREAD

**Discussion by Theletha** [karashiiro/TextToTalk#222] [General]
I really like the elven labs but it gets so expensive on the 100k characters sub u get additional 100k for 30$ but that stacks up to be real expensive in the end. is there cheaper options that isnt the windows tts?

**Comment by johnysandels** [comment]
There is the windows Natural voices you can use, and they work well! the only caveat is the the lexicon files for custom pronunciations don't work yet. You're stuck with the non online ones. If you want to install the voices that work with lexicons, you need to uncheck this option when installing
![Screenshot 2024-07-28 003808](https://github.com/user-attachments/assets/eac9bc59-f3ae-4f7d-a82a-08196f638d8f)

It is a third party solution since using the natural voices is not normally supported in third party programs, and could possibly stop working anytime there's an update
https://github.com/gexgd0419/NaturalVoiceSAPIAdapter

**Reply by saitoh183** [reply]
@johnysandels i had  already installed the missing voices. I installed the app but how do i get it to work with the TTS? I clicked on the missing voices button and installed missing voices and restarted the game but i still just see the voices i had before.

![image](https://github.com/user-attachments/assets/c9437a7f-8c47-4bc2-9617-b7a8705e0227)

**Reply by johnysandels** [reply]
Right now there is a bug where the get more voices button doesn't work. I will look into this at some point but a fix will need to be created and pushed by Kara

**Reply by saitoh183** [reply]
@johnysandels ah so it worked to get the built in ones already but cant be run a second time to get the NAtural voices?
